ain/account.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, Union
+from secrets import token_bytes
+from ain.utils import (
+    privateToPublic,
+    privateToAddress,
+    mnemonicToPrivatekey,
+    keccak
+)
+
+class Account:
+    private_key: str
+    public_key: str
+    address: str
+    
+    def __init__(self, privateKey: Union[bytes, str] = None):
+        if type(privateKey) is bytes:
+            privateKeyBytes = privateKey
+        elif type(privateKey) is str:
+            privateKeyBytes = bytes.fromhex(privateKey)
+        else:
+            raise TypeError('privateKey has invalid type')
+
+        self.private_key = privateKeyBytes.hex()
+        self.public_key = privateToPublic(privateKeyBytes).hex()
+        self.address = privateToAddress(privateKeyBytes)
+
+    @classmethod
+    def fromMnemonic(cls, mnemonic: str, index: int = 0):
+        """
+        Returns an Account with the given mnemonic.
+        """
+        return cls(mnemonicToPrivatekey(mnemonic, index))
+
+    @classmethod
+    def fromEntropy(cls, entropy: str = None):
```

Comment:
suggestion) how about `create()`, since `fromMnemonic()` is getting an account from a mnemonic but this function creates a new account?

Reply:
Changed! PTAL.